.github/workflows/PR.yml

```diff
@@ -83,11 +80,9 @@ jobs:
     - name: Run e2e tests
       id: e2e_tests
       if: github.event_name == 'merge_group'
-      continue-on-error: ${{ matrix.pinecone-plan == 'starter' }}
       env:
         INDEX_NAME: e2e-${{ steps.gen_suffix.outputs.INDEX_NAME_SUFFIX }}
-        PINECONE_ENVIRONMENT: ${{ matrix.pinecone-plan == 'paid' && secrets.PINECONE_ENVIRONMENT_3 || secrets.PINECONE_ENVIRONMENT_4 }}
-        PINECONE_API_KEY: ${{ matrix.pinecone-plan == 'paid' && secrets.PINECONE_API_KEY_3 || secrets.PINECONE_API_KEY_4 }}
+        PINECONE_API_KEY: ${{ secrets.PINECONE_API_KEY_5 }}
```

Comment:
Let's call it simply `PINECONE_API_KEY`?  
The `3` and `4` were used to indicate "pod" VS "serverless", not versions...

Reply:
DONE